README.md

```diff
@@ -4,6 +4,10 @@
   <h2> TUI interface for LLMs written in Rust </h2>
 </div>
 
+<a href="https://repology.org/project/tenere/versions">
+    <img src="https://repology.org/badge/vertical-allrepos/tenere.svg" alt="Packaging status" align="right">
+</a>
```

Comment:
Can we move ti `Installation` section ? Also, maybe change the alignment to be on the left

Reply:
yeah, totally